arwen/host/execution_test.go

```diff
@@ -1486,77 +1486,6 @@ func TestExecution_Mocked_Wasmer_Instances(t *testing.T) {
 	require.Equal(t, expectedVMOutput, vmOutput)
 }
 
-func TestExecution_GasUsed_SingleContract(t *testing.T) {
```

Comment:
why did you delete these tests ?

Reply:
Not deleted, just moved them to `execution_gas_test.go`, and added some more.